cengage/react-magma
# feat(Charts): Add the `Data Visualization` tab with pages `Getting Started`, `Chart Types`, and `Chart Demos`. Add new examples in the storybook.

Issue: #1172
Issue: #1054
Issue: #1055
Issue: #1056

## What I did

Add the `Data Visualization` tab with pages `Getting Started`, `Chart Types`, and `Chart Demos`. Add new examples in the storybook.

## Screenshots:


## Checklist 

## How to test

Open documentation -> Open `Data Visualization` -> Check new pages.
Open storybook -> Open `Charts` folder -> Check new examples.
